Add cluster-port parameter (#1664)
redis/redis-doc: topics/cluster-spec.md

---

Every Redis Cluster node has an additional TCP port for receiving

> incoming connections from other Redis Cluster nodes. This port could be specified in redis.conf file, 
> or it could be obtained by adding 10000 to the data port.  

Reviewer: ```suggestion
incoming connections from other Redis Cluster nodes. This port will be derived by adding 10000 to the data port or it can be specified with the cluster-port config. 
```

* The `currentEpoch` and `configEpoch` fields of the sending node that are used to mount the distributed algorithms used by Redis Cluster (this is explained in detail in the next sections). If the node is a replica the `configEpoch` is the last known `configEpoch` of its master.
* The node flags, indicating if the node is a replica, a master, and other single-bit node information.
* A bitmap of the hash slots served by the sending node, or if the node is a replica, a bitmap of the slots served by its master.
* The sender TCP base port that is the port used by Redis to accept client commands.

> * The cluster port that is the port used by Redis to node-to-node communication.

Reviewer: ```suggestion
* The cluster port that is the port used by Redis for node-to-node communication.
```
